readme.md

```diff
@@ -16,35 +16,49 @@ $ npm install --save hook-std
 const assert = require('assert');
 const hookStd = require('hook-std');
 
-const unhook = hookStd.stdout(output => {
+const promise = hookStd.stdout(output => {
+	promise.unhook();
+
+	assert.strictEqual(output.trim(), 'unicorn');
+});
+
+console.log('unicorn');
+await promise;
+```
+
+Unhook can also be done via callback:
+
+```js
+const promise = hookStd.stdout((output, encoding, unhook) => {
```

Comment:
What do you mean by "Instead, just convert it to a string"? Do you mean to convert the `output` argument to a string before passing it to the `transform` function?